src/Components/ADT3DSceneBuilder/Internal/Elements/Internal/ElementTwinAliasForm.tsx

```diff
@@ -132,7 +129,7 @@ const ElementTwinAliasForm: React.FC = () => {
                         }
                     }}
                 />
-                <TwinPropertySearchDropdown
+                {/* <TwinPropertySearchDropdown
```

Comment:
do you want to remove teh commented section now?

Reply:
so close to a no comments PR :(